### ISSUE TITLE
Make stopwatch log format parser friendly

### ISSUE THREAD

**Issue by rstml** [jalkanen/speed4j#14]
Currently stop("MYTAG", "my message") adds following to the logfile:

```
MYTAG: 29798 us my message
```

This kind of string is really hard to parse (and read). I think it would be handy to have something similar to perf4j format:

```
time[29] tag[MYTAG] message[my message]
```

**Comment by jalkanen** [comment]
Hmm... I didn't think of machine parsers when I designed this. Perhaps it might be better to make it somehow user-defineable rather?

**Comment by rstml** [comment]
user-definable formatting is the most flexible, but will need extra cpu for formatting. i personally wouldn't sacrifice performance in that case and just go with something predefined/static.

**Comment by jalkanen** [comment]
Hmm... Maybe a MachineReadableStopWatch class and a way to configure which StopWatch instances you'll get?

**Comment by rstml** [comment]
Sounds good, but I'd give it more generic name, something like FormattedStopWatch or CustomStopWatch. People may find usage not only for machine parsing.

**Comment by jalkanen** [comment]
Oops, didn't mean to close this. Anyhoo, how does this look? Not sure whether nanoseconds is the right unit here.

**Comment by rstml** [comment]
Looks good. In my case I need ms but more granular timer shouldn't be a problem.
